src/components/Button.jsx

```diff
@@ -0,0 +1,8 @@
+const Button = ({ onClick, children }) => {
```

Comment:
This component is not needed because it has no unique properties. Its styling is handled at the "App.css" and "index.css" l